docs/how-to-use-hipblaslt-offline-tuning.rst

```diff
@@ -0,0 +1,85 @@
+
+.. code-block:: bash
+
+    ./sample_hipblaslt_gemm
+
+- The tuning command will log:
+
+.. code-block:: bash
+
+    hipblaslt-bench --api_method c -m 1024 -n 512 -k 1024 --lda 1024 --ldb 1024 --ldc 1024 --ldd 1024  --stride_a 0 --stride_b 0 --stride_c 0 --stride_d 0  --alpha 1.000000 --beta 1.000000 --transA N --transB N --batch_count 1  --a_type f16_r --b_type f16_r --c_type f16_r --d_type f16_r --scale_type f32_r --bias_type f32_r   --compute_type f32_r 
+
+
+2. Set the environment variable ``HIPBLASLT_TUNING_FILE=<file_name>`` to tune and store the tuning result of the best solution indices for those GEMM problems, where ``<file_name>`` points to the tuning file.
+
+- Please note that the default settings will have the following changes in the tuning environment.
+
+.. code-block:: bash
+
+    --iters |-i <value>             Iterations to run inside timing loop                                                (Default value is: 1000)
+    --cold_iters |-j <value>        Cold Iterations to run before entering the timing loop                              (Default value is: 1000)
+    --algo_method <value>           Use different algorithm search API. Options: heuristic, all, index (still considered all).                  (Default value is: all)
+    --requested_solution <value>    Requested solution num. Set to -1 to get all solutions. Only valid when algo_method is set to heuristic.    (Default value is: -1)
+    --rotating <value>              Use rotating memory blocks for each iteration, size in MB.                          (Default value is: 512)
+
+- After the tuning completes, the expected output is as follows:
+
+.. code-block:: bash
+
+    ./hipblaslt-bench --api_method c -m 64 -n 64 -k 1024 --lda 64 --ldb 1024 --ldc 64 --ldd 64  --stride_a 0 --stride_b 0 --stride_c 0 --stride_d 0  --alpha 1.100000 --beta 0.900000 --transA N --transB N --batch_count 1  --a_type f32_r --b_type f32_r --c_type f32_r --d_type f32_r --scale_type f32_r --bias_type f32_r   --compute_type f32_r
+    
+    Winner: 
+    transA,transB,grouped_gemm,batch_count,m,n,k,alpha,lda,stride_a,beta,ldb,stride_b,ldc,stride_c,ldd,stride_d,a_type,b_type,c_type,d_type,compute_type,scaleA,scaleB,scaleC,scaleD,amaxD,activation_type,bias_vector,bias_type,rotating_buffer,hipblaslt-Gflops,hipblaslt-GB/s,us,soulution_index
+    N,N,0,1,64,64,1024,1.1,64,65536,0.9,1024,65536,64,4096,64,4096,f32_r,f32_r,f32_r,f32_r,f32_r,0,0,0,0,0,none,0,f32_r,512,903.229,55.8608,9.28735,49257
+
+
+3. Set the environment variable ``HIPBLASLT_TUNING_OVERRIDE_FILE=<file_name>`` to load the tuning file and override the default kernel selection with the kernels found, where ``<file_name>`` points to the tuning file.
+
+- For example, user can use ``hisblaslt-bench`` (algo_method: heuristic) to obtain solutions for a problem, which will include the best tuning solution index.
```

Comment:
```suggestion
   For example, a user can use ``hisblaslt-bench`` (with ``algo_method`` set to ``heuristic``) to obtain the solutions for a problem, which include the best tuning solution index.
```

Reply:
Are the solutions below (0, 1, and 2) ranked in order, where 0 is the best solution? Also, is it possible for the user to save the hipblaslt-bench command displayed below and run it again if they want?

Reply:
Yes, the tuning result of the best solution will be placed at position zero.